ci/jenkins/test.sh

```diff
@@ -1154,22 +1157,18 @@ fi
 
 trap clean_antrea EXIT
 if [[ ${TESTCASE} =~ "windows" ]]; then
-    if [[ ${TESTCASE} =~ "containerd" ]]; then
+    if [[ ${TESTCASE} =~ "ovs" ]]; then
```

Comment:
I suggest not changing the TESTCASE name, as it will take effect immediately after the merge, causing the previous branches and docker-related scripts to become unavailable.

Reply:
Maybe we can wait for https://github.com/antrea-io/antrea/pull/5661 to merge ? This change will then become applicable
@XinShuYang